### PR TITLE
docs: add more questions in FAQ.md

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Allen Sun <allensun.shl@alibaba-inc.com>

### Ⅰ. Describe what this PR did

FAQ is very important for end-users or developers to understand the project much better.
We have not sufficient question&answer in Dragonfly before. 

Now I try to add more question&answer in FAQ.md.

If any reviewers or readers find that there should be more concrete questions there in FAQ.md, please do not hesitate to add question&answer in the file within a pull request, or in issue to request maintainers to add.

### Ⅱ. Does this pull request fix one issue?

fix https://github.com/dragonflyoss/Dragonfly/issues/212
fix https://github.com/dragonflyoss/Dragonfly/issues/301 via question ## What is the policy of bandwidth limit in peer network

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
no need.

### Ⅳ. Describe how to verify it
none

### Ⅴ. Special notes for reviews
none

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -41,6 +33,82 @@ At the same time, it also plays the role of peer, which can transfer data betwee
...
+## What is the size of block(piece) when distribution
+
+Dragonfly tries to make block(piece) size dynamically to ensure efficiency.
+
+The size of pieces which is calculated as per the following strategy:
+
+- If file's total size is less than 200MB, then the piece size is `4MB` by default.
+- Otherwise, it equals to `min{ totalSize/100MB + 2 MB, 15MB }`.
+
+## What is the difference between Dragonfly's P2P algorithm and bit-torrent
```
Comment (allencloud) [inline FAQ.md]: Could you help to answer this question? @garfield009 @lowzj 
I think many peopler are curious about the answer.

**Comment by allencloud** [comment]
Do we have any other frequently asked questions? If yes, please comment on this pr or submit prs to add them? @Starnop @zhouhaibing089 @fengzixu @lowzj @garfield009 
Thanks a lot.

**Comment by allencloud** [comment]
> I suggest that we should add related info to the FAQ.md. 

Good idea.  @fengzixu
Now added.

**Comment by fengzixu** [comment]
LGTM

File: FAQ.md [hunk 2]
```diff
@@ -41,6 +40,130 @@ At the same time, it also plays the role of peer, which can transfer data betwee
...
+## What is the size of block(piece) when distribution
+
+Dragonfly tries to make block(piece) size dynamically to ensure efficiency.
+
+The size of pieces which is calculated as per the following strategy:
+
+- If file's total size is less than 200MB, then the piece size is `4MB` by default.
+- Otherwise, it equals to `min{ totalSize/100MB + 2 MB, 15MB }`.
+
+## What is the difference between Dragonfly's P2P algorithm and bit-torrent(BT)
```
Comment (allencloud) [inline FAQ.md]: fix https://github.com/dragonflyoss/Dragonfly/issues/212

**Comment by allencloud** [comment]
LGTM

**Pull request merged**